### DISCUSSION TITLE
How to use RSS with Paige ?

### DISCUSSION THREAD

**Discussion by ghost** [willfaught/paige#87] [Q&A] [answered]
After a site is published using Paige theme, how would an end user subscribe to RSS ?

**Comment by willfaught** [comment, marked as answer]
Assuming you enabled the rss output, every list page has an rss.xml file, e.g. /rss.xml, /blog/rss.xml, /categories/rss.xml, /categories/foo/rss.xml. When you give https://yoursite.com to most RSS readers, they will automatically try https://yoursite.com/rss.xml.
